docs/content/getting-started/code-generation.md

```diff
@@ -138,33 +136,33 @@ $ hof gen data.cue schema.cue -T template.txt -T =debug.yaml -O out/
 
 ### Watching for Changes
 
-Use `-w`/`--watch` to watch for changes and re-render output.
+Use `-w`/`--watch` to observe for changes and re-render output.
 
 {{<codeInner title="> terminal" lang="sh">}}
 $ hof gen data.cue schema.cue -T template.txt -T debug.yaml -O out/ --watch
 {{</codeInner>}}
 
-There are extra watch flags if automatic detection doesn't fully work.
+There are extra watch flags if automatic detection doesn't entirely work.
 
 
 ## On Using CUE
 
 <br>
 
-`hof`'s inputs are `cue`'s inputs, or "CUE entrypoints".
+`hof`'s inputs are `cue`'s inputs, or "CUE entry points".
 
 The inputs hold CUE values, which can be intermixed with your data to
 apply schemas, enrich the data, or transform before rendering.
-When running commands, the CUE entrypoints are combined into one large CUE value.
-The final data passed to a template must be concrete, or fully specified.
-This means the value needs to be like JSON data before template rendering will accept them.
+When running commands, the CUE entry points are combined into one considerable CUE value.
+The final data passed to a template must be concrete or fully specified.
+This means the value needs to be like JSON data before template rendering accepts them.
 As you will see, `hof` provides you flexibility and control
 for how the CUE values are selected, combined, and mapped to templates.
 
-We keep parity with `cue` so tooling in the wider ecosystem
-still works on our inputs and reduces context switching costs.
+We keep parity with `cue` so tooling on the broader ecosystem
```

Comment:
on -> from?

Reply:
If the intention is to say that we are keeping the input format compatible with cue, in order to ensure that tools from the broader ecosystem can still work on their inputs, then the first sentence is more appropriate. - `from` 

If the intention is to say that the input format is already compatible with the broader ecosystem, then the second sentence seems to be more appropriate. - `on`

Reply:
i've updated to `from` 🔨